### ISSUE TITLE
Will spits out a bunch of errors shortly after startup

### ISSUE THREAD

**Issue by wontonst** [skoczen/will#326]
The following error is encountered, but did not appear to affect the "hello" plugin.

```
Tue, 05 Dec 2017 09:25:36 [CRITICAL] Error handling incoming event <deleted this string to hide personal info>:
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/site-packages/will/backends/io_adapters/base.py", line 43, in handle_incoming_event
    m = self.normalize_incoming_event(event)
  File "/usr/local/lib/python3.4/site-packages/will/backends/io_adapters/hipchat.py", line 644, in normalize_incoming_event
    if sender.id == self.me.id:
AttributeError: 'NoneType' object has no attribute 'id'
```

This issue occurred together with #322

**Comment by skoczen** [comment]
(Quick paste-on-all-open-issues update:)

**TL;DR**: New Will release EOM. Also, you're awesome.

**Full Update:** This may be TMI for you, and if so, cool, the thing you probably care about is above.  But given the total radio silence on my part - and the fact that I've always tried to create a community here where we got to be actual people instead of avatars and PRs, I wanted to give a bit more context.

As you may be aware from an earlier AFK, I got married in December, which was great, but burned the month from a Will perspective.  I planned to get back on it at the start of January, but instead found myself selling our house, and simultaneously hit with a surprisingly severe bout of depression.  The time since Jan 1 has been sort of a blur.  A dark blur, but a blur more than anything.   I saw the PRs and comments coming in, but didn't have the energy or time to do anything about them.

Things seem to be clearing up on both counts now, and I'm _really_ looking forward to getting these fixes in, catching up on bug reports, and being active in improving Will again. :)

With that, I'm aiming to get all outstanding PRs bundled into a release, and all issues replied to by the end of February.

Most of all, thanks for your contribution to Will, your patience with an imperfect, quite human project maintainer, and for continuing to come back, engage, and spend some of the precious seconds you get on earth making this project better.

I appreciate you.  Thank you.

**Comment by wontonst** [comment]
@skoczen glad you're feeling better. Will is a great project and I'm inspired that you have the fortitude to fight through your issues and carry on.

**Comment by skoczen** [comment]
This will be fixed in 2.1.1!

**Comment by skoczen** [comment]
(Also, thanks! :) )

**Comment by skoczen** [comment]
This has been fixed, and released in 2.1.1!
